lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -5186,15 +5200,15 @@ public String reorder() {
 		String[] split = split(order, " ");
 
 		// make sure nothing is duplicated. I know it shouldn't be, but
-		// I saw the Fluid reorderer get confused once.
+		/*/ I saw the Fluid reorderer get confused once.
 		Set<String> used = new HashSet<>();
 		for (int i = 0; i < split.length; i++) {
 			if (!used.add(split[i].trim())) {
 				log.warn("reorder: duplicate value");
 				setErrMessage(messageLocator.getMessage("simplepage.reorder-duplicates"));
 				return "failed"; // it was already there. Oops.
 			}
-		}
+		}*/	//this is currently commented out because it was getting incorrectly triggered on use of the new site-selecting version of Add Items From Another Page. I believe this is happening because of a bug in the way new items are marked in ReorderProducer.fillComponents, which leads some to get marked incorrectly. If we want to come back to it, I don't want this existing duplicate-checking logic to get lost.
```

Comment:
It maybe related to SAK-46773

Reply:
Just un-commented it and pushed again. David relayed to me that this issue could be fixed now by some recent work you've done, so I put it back and things seem to be working fine on my end.